ReactViewControl/ReactViewRender.cs

```diff
@@ -53,7 +53,10 @@ internal partial class ReactViewRender : IChildViewHost, IDisposable {
             DefaultStyleSheet = defaultStyleSheet;
             PluginsFactory = initializePlugins;
             EnableDebugMode = enableDebugMode;
-            DevServerUri = devServerUri;
+
+            if (moduleDependenciesProvider != null) {
```

Comment:
we can remove the null check, since it will be null anyway if we don't the assign